render.js

```diff
@@ -0,0 +1,16 @@
+var render = (function() {
+
+  var MovieInfo = function(response) {
```

Comment:
I would suggest maybe calling it renderMovieInfo... but this is open to discussion.